pkg/controller/restore_finalizer_controller.go

```diff
@@ -268,6 +273,12 @@ func (ctx *finalizerContext) patchDynamicPVWithVolumeInfo() (errs results.Result
 
 	var pvWaitGroup sync.WaitGroup
 	var resultLock sync.Mutex
+	var pvPatchTimeout = PVPatchMaximumDuration
+
+	if ctx.resourceTimeout > 0 {
```

Comment:
I don't think this is necessary, because we always have a default value(10min) for `resourceTimeout`, so it never <= 0.
So we can just delete `PVPatchMaximumDuration` and use `ctx.resourceTimeout` directly, which makes the code simpler.

Reply:
Done, updated the PR